libraries/Wire/Wire_nRF51.cpp

```diff
@@ -55,7 +55,7 @@ void TwoWire::begin(void) {
                                 | ((uint32_t)GPIO_PIN_CNF_DRIVE_S0D1       << GPIO_PIN_CNF_DRIVE_Pos)
                                 | ((uint32_t)GPIO_PIN_CNF_SENSE_Disabled   << GPIO_PIN_CNF_SENSE_Pos);
 
-  _p_twi->FREQUENCY = TWI_FREQUENCY_FREQUENCY_K100;
+  _p_twi->FREQUENCY = (TWI_FREQUENCY_FREQUENCY_K100 << TWI_FREQUENCY_FREQUENCY_Pos);
```

Comment:
Ok, we'll have to do another pass through all the other source in this repo for it to be consistent - but that's another PR :)